src/router/index.ts

```diff
@@ -12,12 +13,37 @@ const router = createRouter({
     {
       path: '/about',
       name: 'about',
+      meta: { public: true },
```

Comment:
Each route we want public (accessible to all), should have this meta property added.

When `public: true` is omitted, users will be redirect to the sign-in page.